### ISSUE TITLE
Image quality degradation with Flux kontext

### ISSUE THREAD

**Issue by kabalao** [black-forest-labs/flux#455]
Run the code:
```
import torch
from diffusers import FluxKontextPipeline
from diffusers.utils import load_image

pipe = FluxKontextPipeline.from_pretrained("black-forest-labs/FLUX.1-Kontext-dev", torch_dtype=torch.bfloat16)
pipe.to("cuda")

input_image = load_image("https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/cat.png")

image = pipe(
  image=input_image,
  prompt="Add a hat to the cat",
  guidance_scale=2.5
).images[0]
image.save("kontext.png")
```
Please see the generated image, looks the image quality is bad. Any clue?
![Image](https://github.com/user-attachments/assets/619402e4-b240-4104-84b5-ef2857f95f4e)

![Image](https://github.com/user-attachments/assets/242c0093-adea-44fb-9e41-faaecad5f2eb)

**Comment by OwalnutO** [comment]
Same problem. The inference pipeline may contain some crop and padding operations. I guess this is the main reason.

**Comment by zhaoqi571436204** [comment]
How to solve it?

**Comment by Iriya99** [comment]
Same question. I noticed that the generated images may be displaced or scaled
